rbac/server/api/users.py

```diff
@@ -226,6 +292,34 @@
 
 # TODO: Change → api/users/<next_id>
 @USERS_BP.put("api/users/update")
+@doc.summary("Update the details associated with a user.")
+@doc.description("Update the details associated with a user. Restricted to NEXT standalone mode.")
+@doc.consumes(
+    doc.JsonBody(
+        {
+            "next_id": str,
+            "name": str,
+            "username": str,
+            "email": str,
+        }
+    ),
+    location="body",
+    content_type="application/json",
+    required=True,
+)
+@doc.produces(
+    {
+        "message": "User information was successfully updated."
+    },
+    content_type="application/json",
```

Comment:
For future reference, you won't need to specify the content_type here. I added a configuration in one my PRs to automatically set the returned content type

Reply:
Oh cool, should I remove them then since it's a default value?

Reply:
You can leave it here - seems you already have them.